### PR TITLE
Adding the REDASA Open Dataset

### PR DESCRIPTION
**Pull request by oserban**
*Description of changes:* Adding the REDASA Open Dataset

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 897ec4ca3707800818e01ffd2b2c840b43f20623
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e1391bc3-7c01-4f67-8f82-fde8527ed83e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: aadafeecc50100031b61a3196c4328434819704f
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=0e357a46-cc85-4e09-ad61-28c6399f64a6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 32e98e6e88b40650f816fdd7007c56b5a3d12a90
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=0d559b2f-a98c-434c-9dad-fa9ec5d49b7e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 9abd69aebf086a8dd614505028d16092541c208c
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=ba562157-9715-4c89-b333-9ed527e023a3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 947176f9e6a8accb9c11093924caf15c7e6cd2cf
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=1ca8c206-8c58-41f2-a85e-11c5b0e72ef3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: ce8ab9d8d68d4557569bd992767c9d032dbbb9b2
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6c63204c-853f-4124-a382-b38e2c3ed4cc%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/redasa-covid-data.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Resources:
+  - Description: This is the raw data repository containing a common crawl of CORD-19 papers and other sources identified by the REDASA Project.
+    ARN: arn:aws:s3:::pansurg-curation-raw-open-data
+    Region: eu-west-2
+    Type: S3 Bucket
+  - Description: For all the questions curated during the REDASA project, we created a Kendra index. The documents available in this S3 bucket were surfaced by the Kendra index as being relevant to the research medical question.
+    ARN: arn:aws:s3:::pansurg-curation-workflo-kendraqueryresults50d0eb-open-data
+    Region: eu-west-2
+    Type: S3 Bucket
+  - Description: An S3 bucket that contains the final curation data in GroundTruth format
```
Comment (chueatwork) [inline datasets/redasa-covid-data.yaml:28]: @oserban do you mean SageMaker GroundTruth manifest?

File: datasets/redasa-covid-data.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - COVID-19
+  - coronavirus
+  - life sciences
+  - information retrieval
+  - natural language processing
+  - text analysis
+License: CC-BY-4.0
+Resources:
+  - Description: This is the raw data repository containing a common crawl of CORD-19 papers and other sources identified by the REDASA Project.
+    ARN: arn:aws:s3:::pansurg-curation-raw-open-data
```
Comment (chueatwork) [inline datasets/redasa-covid-data.yaml:21]: @oserban currently I cannot list contents for any of these buckets. Did you use our CloudFormation template to generate these? If no, we will want to alter the bucket policy to make these world-readable. We'll also want to make sure you have [bucket logs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-server-access-logging.html) and [CloudWatch usage metrics](https://docs.aws.amazon.com/AmazonS3/latest/userguide/configure-request-metrics-bucket.html) enabled on each bucket.

Comment (oserban) [reply datasets/redasa-covid-data.yaml:21]: @chueatwork I enabled public access to all the resources, plus the bucket logs and cloudwatch metrics. Please let me know if you can list the contents.

File: datasets/redasa-covid-data.yaml (hunk 1 above)
Comment (oserban) [reply datasets/redasa-covid-data.yaml:28]: I wanted to say that the format of the file is whatever SageMaker GroundTruth outputs at the end of an annotation task, but if you feel it needs a more detailed description I can do that as well.

Comment (chueatwork) [reply datasets/redasa-covid-data.yaml:28]: Ah, I see. I think we can say SageMaker GroundTruth manifests, but I'll double check and make the edit myself if need be.

**Pull request merged**